src/SwedbankPay.Sdk.Infrastructure/PaymentInstruments/MobilePay/MobilePayPaymentsResource.cs

```diff
@@ -0,0 +1,35 @@
+﻿using SwedbankPay.Sdk.Extensions;
+using SwedbankPay.Sdk.PaymentInstruments.MobilePay;
+using SwedbankPay.Sdk.Payments.MobilePayPayments;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.Payments
+{
+    public class MobilePayPaymentsResource : ResourceBase, IMobilePayResource
+    {
+        public MobilePayPaymentsResource(HttpClient httpClient) : base(httpClient)
+        {
+        }
+
+        public async Task<IMobilePayPaymentResponse> Get(Uri id, PaymentExpand paymentExpand = PaymentExpand.None)

```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)